components/AddTaskCard.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="card" v-if="showForm">
+    <textarea v-model="todoText"></textarea>
+    <div class="card-footer">
+      <button @click="addTask">{{ $t("Add Task") }}</button>
+      <span class="delete-icon" @click="clearField">
+        <img :src="deleteIcon" />
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from "vuex";
+export default {
+    data() {
+    return {
+      todoText: "",
+      deleteIcon: require("@/assets/img/delete.svg"),
+    };
+  },
+  computed: {
+    ...mapState(["showForm"]),
```

Comment:
As `showForm` boolean is specific to this component, we can keep this state coupled to this component rather than making this global.

Reply:
resolved